src/EnvConfig.php

```diff
@@ -18,63 +18,29 @@ class EnvConfig implements SiteConfig
     public const PRODUCTION = 'production';
     public const STAGING = 'staging';
 
-    private const ENV_ALIASES = [
-        'local' => self::LOCAL,
-        'development' => self::DEVELOPMENT,
-        'dev' => self::DEVELOPMENT,
-        'develop' => self::DEVELOPMENT,
-        'staging' => self::STAGING,
-        'stage' => self::STAGING,
-        'preprod' => self::STAGING,
-        'pre-prod' => self::STAGING,
-        'pre-production' => self::STAGING,
-        'test' => self::STAGING,
-        'uat' => self::STAGING,
-        'production' => self::PRODUCTION,
-        'prod' => self::PRODUCTION,
-        'live' =>  self::PRODUCTION,
-    ];
```

Comment:
This was replaced by using `wp_get_environmen_type()`. Will this affect current systems which are using the non-defined WordPress env types (local, development, staging, production)?

Reply:
The replaced code was still reducing the env names to the one supported by WP. Because after being obtained, the [environment was "normalized"](https://github.com/inpsyde/wp-app-container/blob/c2a09ac52fbafb4a6083998765947b87b8687725/src/EnvConfig.php#L179) to [one of the class constants](https://github.com/inpsyde/wp-app-container/blob/master/src/EnvConfig.php#L16-L19) which are exactly the values returned by `wp_get_environmen_type()`.

Moreover, if you were using a WP version that has `wp_get_environment_type()` available (and considering that's 5.5+ chances are you are) [**the class was _already_ using `wp_get_environment_type()`**](https://github.com/inpsyde/wp-app-container/blob/c2a09ac52fbafb4a6083998765947b87b8687725/src/EnvConfig.php#L169-L170).

In short, this change is just removing a good amount of complexity that was there just to account for WP < 5.5, which we don't want to care anymore...